OpenRA.Platforms.Default/Sdl2PlatformWindow.cs

```diff
@@ -156,16 +156,17 @@ public GLProfile[] SupportedGLProfiles
 					.ToArray();
 
 				if (!supportedProfiles.Any())
-					throw new InvalidOperationException("No supported OpenGL profiles were found.");
+					throw new InvalidOperationException($"No supported OpenGL profiles were found: {SDL.SDL_GetError()}");
 
 				profile = supportedProfiles.Contains(requestProfile) ? requestProfile : supportedProfiles.First();
 
 				// Note: This must be called after the CanCreateGLWindow checks above,
 				// which needs to create and destroy its own SDL contexts as a workaround for specific buggy drivers
-				SDL.SDL_Init(SDL.SDL_INIT_VIDEO);
-				SetSDLAttributes(profile);
+				if (SDL.SDL_Init(SDL.SDL_INIT_VIDEO) != 0)
```

Comment:
I thought that this and the string formatting changes were worthwhile, I just wanted the change above unstaged!

Reply:
I see. However, the original error was only visible during profile detection, so removing that made everything futile. I added the per profile SDL error log now.